include/cuco/detail/static_multiset/static_multiset.inl

```diff
@@ -412,6 +401,80 @@ static_multiset<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>
     stream);
 }
 
+template <class Key,
+          class Extent,
+          cuda::thread_scope Scope,
+          class KeyEqual,
+          class ProbingScheme,
+          class Allocator,
+          class Storage>
+template <class InputProbeIt, class OutputProbeIt, class OutputMatchIt>
+std::pair<OutputProbeIt, OutputMatchIt>
+static_multiset<Key, Extent, Scope, KeyEqual, ProbingScheme, Allocator, Storage>::retrieve(
+  InputProbeIt first,
+  InputProbeIt last,
+  OutputProbeIt output_probe,
+  OutputMatchIt output_match,
+  cuda::stream_ref stream) const
+{
+  return this->impl_->retrieve(
```

Comment:
```suggestion
  return impl_->retrieve(
```

I'm not 100% sure why this `retrieve` function shows up in the changes. The git diff looks a bit broken somehow.